app/assets/templates/static_pages/index.html

```diff
@@ -1,43 +1,32 @@
-<nav class="navbar">
-  <div class="container-fluid wrap">
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <a class="navbar-brand" href="/#/">Jimmy This</a>
-    </div>
-    <div id="navbar" class="collapse navbar-collapse">
-      <ul class="nav navbar-nav navbar-right">
-        <li ng-hide="loggedIn"><a href="/#/login">Log In</a></li>
-        <li ng-hide="loggedIn"><a href="/#/signup" class="btn btn-primary">Join Today</a></li>
-        <li class="dropdown" ng-show="loggedIn">
-          <a href="/#/" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Logged In <span class="caret"></span></a>
-          <ul class="dropdown-menu">
-            <li><a href="/#/account">Account</a></li>
-            <li><a href="/#/services">Services</a></li>
-            <li><a href="/#/" ng-click="logout()">Log Out</a></li>
-          </ul>
-        </li>
-      </ul>
-    </div> 
-  </div>
-</nav>
+<navbar></navbar>
```

Comment:
if all the views have a navbar... shouldn't you add it to https://github.com/WDI-HK-8/jimmythis/blob/development/app%2Fviews%2Flayouts%2Fapplication.html.erb?